gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/platform/configuration/ConfigurationResource.java

```diff
@@ -85,4 +86,12 @@ public void getSpelGrammar(@Suspended final AsyncResponse response) {
             .subscribe(response::resume, response::resume);
     }
 
+    @GET
+    @Path("users/emailRequired")
```

Comment:
Path has to be lowercase: "users/email-required"

Shouldn't we think about more generic configuration endpoint for easy adaptability in the future? What do you think?

Reply:
I agree on both points.
The more general endpoint, might be a good idea (we could then just load it once in `main.ts`, together with `constants.json` and `build.json`), but I don't think we currently have enough configuration that needs such an endpoint to know how to best structure it.
I think for now it's good enough that these few config elements are grouped under a single resource.